orders/models.py

```diff
@@ -0,0 +1,56 @@
+from django.db import models
+
+class Cart(models.Model):
+    quantity     = models.IntegerField()
+    user         = models.ForeignKey("users.User",on_delete=models.CASCADE,related_name="cart")
+    product      = models.ForeignKey('products.Product',on_delete=models.CASCADE,related_name='cart')
+    created_at   = models.DateTimeField(auto_now_add=True)
+    updated_at   = models.DateTimeField(auto_now=True)
+    
+    class Meta:
+        db_table = 'carts'
+        
+class Wishlist(models.Model):
+    user         = models.ForeignKey("users.User",on_delete=models.CASCADE,related_name='wishlist')
+    product      = models.ForeignKey('products.Product',on_delete=models.CASCADE,related_name='wishlist')
+    created_at   = models.DateTimeField(auto_now_add=True)
+    updated_at   = models.DateTimeField(auto_now=True)
+    
+    
+    class Meta:
+        db_table = 'wishlists'
+        
+class OrderStatus(models.Model):
+    order_status = models.CharField(max_length=50)
+    created_at   = models.DateTimeField(auto_now_add=True)
+    
+    class Meta:
+        db_table = 'order_statuses'
+
+class Order(models.Model):
+    user         = models.ForeignKey('users.User',on_delete=models.CASCADE,related_name='order')
+    order_status = models.ForeignKey('OrderStatus',on_delete=models.CASCADE,related_name='order')
+    created_at   = models.DateTimeField(auto_now_add=True)
+    updated_at   = models.DateTimeField(auto_now=True)
+    
+    class Meta:
+        db_table = 'orders'
+        
+class OrderItemStatus(models.Model):
+    order_item_status = models.CharField(max_length=50)
+    created_at        = models.DateTimeField(auto_now_add=True)
+    
+    class Meta:
+        db_table = 'order_item_statuses'
+
+class OrderItem(models.Model):
+    product           = models.ForeignKey('products.Product',on_delete=models.CASCADE,related_name='order_item')
+    order             = models.ForeignKey('Order',on_delete=models.CASCADE,related_name='order_item')
+    order_item_status = models.ForeignKey('OrderItemStatus',on_delete=models.CASCADE,related_name='order_item')
```

Comment:
`related_name`은 복수로 선언해주셔야 합니다. order에서 order_item 정보를 볼러올 때, `order.order_items.all()` 코드로 불러오게 됩니다. 이때, 복수의 객체를 불러오기 때문에 `order_items` 복수로 불러오도록 만들어주셔야 합니다-!

```suggestion
    product           = models.ForeignKey('products.Product',on_delete=models.CASCADE,related_name='order_items')
    order             = models.ForeignKey('Order',on_delete=models.CASCADE,related_name='order_items')
    order_item_status = models.ForeignKey('OrderItemStatus',on_delete=models.CASCADE,related_name='order_items')
```

Reply:
many to many field에 있는 related name도 복수로 선언해줘야하나요?